modules/browser/BrowserCapabilities.js

```diff
@@ -219,6 +219,17 @@ export default class BrowserCapabilities extends BrowserDetection {
         return !this.isFirefox();
     }
 
+    /**
+     * Returns true if the browser supports track based statistics for the local video track. Otherwise,
+     * track resolution and framerate will be calculated based on the 'outbound-rtp' statistics.
+     * @returns {boolean}
+     */
+    supportsTrackBasedStats() {
+        return (this.isChromiumBased() && this.isVersionLessThan(112))
+            || this.isWebKitBased()
+            || this.isFirefox();
```

Comment:
I don't see track stats in Firefox? Sadly it does not have framesPerSecond in outbound-rtp but it does have framesSent at least which allows calculating

Reply:
They are working on Safari too so I removed the check for that as well.